.gitignore

```diff
@@ -37,6 +37,7 @@ nim_status_client.log
 *.csv
 .flatpak-builder
 /ui/StatusQ/sandbox/android/.gradle/
+.qmake_previous
```

Comment:
It's used to track the Qt version used for the last build, to clean up some artifacts when switching between versions

Reply:
Nice addition, it annoyed me too :)